error: [autil_vec_get] Index out of bounds
